### PR TITLE
fix typo & clarify test names

### PR DESCRIPTION
**Pull request by Nurou**
## Description

- Fixed the spelling of JUnit from junit -> JUnit to create a trivial PR.
- Clarified naming of some tests to better describe what they're doing. The rest of the tests should probably also be named more descriptively. This makes it easier for other people (such as myself) to figure out what's going on without having to puzzle through the code. 

## How to test:

Test can be run as normal. Changes only affect naming.

### CODE REVIEW

**Review by lambdakilo — APPROVED** [review]
Tests pass with the refactoring - also in this project we have the convention of

Programmer 1 creates the PR
Programmer 2 approves the CI passed PR
Programmer 1(@Nurou ) merges the PR
that being said feel free to merge your PR whenever you feel like it

**Comment by Nurou** [comment]
> Tests pass with the refactoring - also in this project we have the convention of
> 
> Programmer 1 creates the PR
> Programmer 2 approves the CI passed PR
> Programmer 1(@Nurou ) merges the PR
> that being said feel free to merge your PR whenever you feel like it

I can't merge it since I don't have write access to the repo.

**Comment by lambdakilo** [comment]
> I can't merge it since I don't have write access to the repo.

Merging this commit manually due to an exception in the convention.

**Pull request merged**
